src/libraries/System.ComponentModel.Primitives/src/ILLink/ILLink.LinkAttributes.xml

```diff
@@ -0,0 +1,55 @@
+<linker>
+  <assembly fullname="System.ComponentModel.Primitives">
+    <type fullname="System.ComponentModel.BrowsableAttribute">
+      <attribute internal="RemoveAttributeInstances" />
+    </type>
+    <type fullname="System.ComponentModel.CategoryAttribute">
+      <attribute internal="RemoveAttributeInstances" />
+    </type>
+    <type fullname="System.ComponentModel.DescriptionAttribute">
+      <attribute internal="RemoveAttributeInstances" />
+    </type>
+    <type fullname="System.ComponentModel.DesignerAttribute">
```

Comment:
This one is used by the winforms runtime to display the designer of a component.

Reply:
Never mind, I see this file is only included for mobile targets.